src/coreclr/tools/aot/DependencyGraphViewer/Program.cs

```diff
@@ -271,21 +383,29 @@ static class Program
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
+        ///
+        
         [STAThread]
-        static int Main()
+        static int Main(string[] args)
         {
-            // Today you have to be Admin to turn on ETW events (anyone can write ETW events).   
-            if (!(TraceEventSession.IsElevated() ?? false))
+            string argPath = string.Empty;
+            if (args.Length == 0)
             {
-                MessageBox.Show("To turn on ETW events you need to be Administrator, please run from an Admin process.");
-                return -1;
+                argPath = "c:\\Users";
             }
+            else
+            {
+                argPath = args[0];
+            }
+            
+            // Today you have to be Admin to turn on ETW events (anyone can write ETW events). If user elects to use ETW events, they must be Admin.
```

Comment:
Good comment. Is there a better place for it?

Reply:
yep! moved